packages/mongodb-chatbot-evaluation/package.json

```diff
@@ -42,7 +42,8 @@
     "lint": "eslint ./src --ext ts --report-unused-disable-directives",
     "lint:fix": "npm run lint -- --fix && prettier ./src --check --write",
     "release": "release-it",
-    "test": "jest --forceExit"
+    "test": "jest --forceExit",
+    "shuffle": "node ./build/generate/shuffleQuizData.js"
```

Comment:
What's this script? I don't see the corresponding file anywhere

Reply:
oh we can remove that, i made that as a 1-off when creating quiz question dataset

Reply:
removed. closing comment